packages/api-v2/src/student/student.controller.ts

```diff
@@ -60,26 +62,31 @@ export class StudentController {
       throw new BadRequestException();
     }
 
-    return updateResult;
+    const student = await this.studentService.findByUuid(uuid, true);
+
+    if (!student) {
+      throw new BadRequestException();
+    }
+
+    return student;
```

Comment:
we should write tests for backend asap (not this PR tho). @justineluo is there a spot in the timeline for implementing cypress?

Reply:
Ooh, we could have integration tests that hit the endpoints using supertest written pretty quick since Nest is already setup with supertest. I normally use cypress for e2e testing from the frontend but if cypress has a ton of benefits then we can go with that

Reply:
supertest would be good too! mostly anything, so we have some sort of a spec for the api